packages/e2e/features/ui/components/authenticator/gen2/sign-up-with-phone.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Sign Up with Phone 
+
+  Create a new user in the Amazon Cognito UserPool by passing the new user’s phone number and password.
+
+  Background:
+    Given I'm running the example "ui/components/authenticator/gen2/sign-up-with-phone/"
+    Then I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.SignUp" } }' with fixture "sign-up-with-phone"
+
+  @angular @react @vue @react-native
```

Comment:
Do you know if cucumber has a way to share fragments of feature files? Might be something to look into so we don't have to copy the entire files.

Reply:
I can check but likely no way. Even though we can change the route, I am afraid we would have to copy the same scenarios for gen2 in the file